processing/src/main/java/io/druid/query/extraction/MapLookupExtractor.java

```diff
@@ -43,12 +43,16 @@
 {
   private final Map<String, String> map;
 
+  private final boolean isOneToOne;
+
   @JsonCreator
   public MapLookupExtractor(
-      @JsonProperty("map") Map<String, String> map
+      @JsonProperty("map") Map<String, String> map,
+      @JsonProperty("isOneToOne") boolean isOneToOne
```

Comment:
Are there any tests included that use MapLookupExtractor with isOneToOne == true?